### ISSUE TITLE
Player List Data

### ISSUE THREAD

**Issue by Cooldude2606** [explosivegaming/factorio-olympics#36]
**Feature overview**
Name: Player List Data
TL;DR: The player list should be able to show some extra data, such as deaths or crashes.

**Details**
The player list currently only shows the amount of time that a player has been online, this doesnt really have much use on a mini game server and should be replaced. A suggested replacement is to allow mini games to customise what data is shown here, such as deaths, crashes, progress, etc.

**Possible methods**
This would a function call from the mini games to the player list which would define how to get this extra data for each player. This function would take a player as a parameter and returns the string that should be shown next to the players name. This could be extented to also handle the tooltip.
